cpp/include/mcts/Node.hpp

```diff
@@ -141,6 +142,7 @@ class Node {
     float raw_policy_prior = 0;
     float adjusted_policy_prior = 0;
     float child_V_estimate = 0;  // network estimate of child-value for current-player
+    float chance_prob = 0;  // probability of this edge being chosen in a chance node
```

Comment:
Add a comment above this line saying that we could overload `raw_policy_prior` to store the chance prob, to save space.

(That might warrant renaming the variable to something more appropriate for the overloaded usage.)